components/ui/button.tsx

```diff
@@ -9,15 +9,18 @@ const buttonVariants = cva(
   {
     variants: {
       variant: {
-        default: "bg-primary text-primary-foreground hover:bg-primary/90",
-        destructive:
-          "bg-destructive text-destructive-foreground hover:bg-destructive/90",
+        // Built-in variants
```

Comment:
Remove commented out code. No need to keep this in the code base. We can always look at the history to view unused code.

Reply:
Done. Committed and will be pushed when all other changes are complete.